src/runtime/offers-flow.js

```diff
@@ -74,34 +109,47 @@ export class OffersFlow {
    * @return {!Promise}
    */
   start() {
-    // Start/cancel events.
-    this.deps_.callbacks().triggerFlowStarted(SubscriptionFlows.SHOW_OFFERS);
-    this.activityIframeView_.onCancel(() => {
-      this.deps_.callbacks().triggerFlowCanceled(SubscriptionFlows.SHOW_OFFERS);
-    });
+    if (this.activityIframeView_) {
+      // so no error is thrown if offers skipped
```

Comment:
Please clarify the comment (I'm not sure what you're trying to say here)

Reply:
Done, hopefully clearer now.